logging/exportlogs/exportlogs.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample exportlogs lists, creates, updates, and deletes log sinks.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"google.golang.org/api/iterator"
+
+	"cloud.google.com/go/logging/logadmin"
+)
+
+func main() {
+	if len(os.Args) == 2 {
+		usage("Missing command.")
+	}
+	if len(os.Args) != 3 {
+		usage("")
+	}
+
+	projID := os.Args[1]
+	command := os.Args[2]
+
+	// [START create_logging_client]
+	ctx := context.Background()
+	client, err := logadmin.NewClient(ctx, projID)
+	if err != nil {
+		log.Fatalf("logadmin.NewClient: %v", err)
+	}
+	// [END create_logging_client]
+
+	switch command {
+	case "list":
+		log.Print("Listing log sinks.")
+		sinks, err := listSinks(client)
+		if err != nil {
+			log.Fatalf("Could not list log sinks: %v", err)
+		}
+		for _, sink := range sinks {
+			fmt.Printf("Sink: %v\n", sink.ID)
+		}
+	case "create":
+		if err := createSink(client); err != nil {
+			log.Fatalf("Could not create sink: %v", err)
+		}
+	case "update":
+		if err := updateSink(client); err != nil {
+			log.Fatalf("Could not update sink: %v", err)
+		}
+	case "delete":
+		if err := deleteSink(client); err != nil {
+			log.Fatalf("Could not delete sink: %v", err)
+		}
+	default:
+		usage("Unknown command.")
+	}
+}
+
+func listSinks(client *logadmin.Client) ([]*logadmin.Sink, error) {
+	// [START list_log_sinks]
+	ctx := context.Background()
+
+	var sinks []*logadmin.Sink
+	it := client.Sinks(ctx)
+	for {
+		sink, err := it.Next()
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			return nil, err
+		}
+		sinks = append(sinks, sink)
```

Comment:
`sinks = append(sinks, sink.ID)` perhaps?

return a `[]string`

Demonstrates a small amount of manipulation